x-pack/plugin/spatial/src/main/java/org/elasticsearch/xpack/spatial/search/aggregations/bucket/geogrid/GeoHexCellIdSource.java

```diff
@@ -84,6 +91,24 @@ protected int advanceValue(org.elasticsearch.common.geo.GeoPoint target, int val
         };
     }
 
+    // package private for testing
+    static LatLonBounds getGeoBounds(CellBoundary cellBoundary) {
+        final LatLonBounds bounds = new LatLonBounds();
+        org.apache.lucene.spatial3d.geom.GeoPoint A = getGeoPoint(cellBoundary.getLatLon(cellBoundary.numPoints() - 1));
```

Comment:
A bit non-standard to use capital variable names.

Reply:
change it for start / end.

Reply:
Sounds good. With this pattern I normally use `current` and `previous`, so the last line of the loop contains the statement `previous = current;`